### ISSUE TITLE
Wrong class name generated for google.protobuf.Empty message

### ISSUE THREAD

**Issue by zarianec** [spiral/php-grpc#11]
### Problem
I'm trying to use `google.protobuf.Empty` message in my rpc declaration but it's generated with wrong name - `PBEmpty` instead of `GPBEmpty`.

Here is generated message in protobuf repository - [GPBEmpty](https://github.com/protocolbuffers/protobuf/blob/master/php/src/Google/Protobuf/GPBEmpty.php)

### How to reproduce
*Test `service.proto`:*
```
syntax = "proto3";

package test;

import "google/protobuf/empty.proto";

service Service {
    rpc Test (google.protobuf.Empty) returns (google.protobuf.Empty) {
    }
}
```

*Expected output:*
```
<?php
# Generated by the protocol buffer compiler (spiral/php-grpc). DO NOT EDIT!
# source: service.proto
 
namespace Test;

use Spiral\GRPC;
use Google\Protobuf;

interface ServiceInterface extends GRPC\ServiceInterface
{
    // GRPC specific service name.
    public const NAME = "test.Service"; 

    /**
    * @param GRPC\ContextInterface $ctx
    * @param Protobuf\GPBEmpty $in
    * @return Protobuf\Any
    *
    * @throws GRPC\Exception\InvokeException
    */
    public function Test(GRPC\ContextInterface $ctx, Protobuf\GPBEmpty $in): Protobuf\Any;
}

```

**Comment by wolfy-j** [comment]
Hmmm, I might be using invalid prefix... I was under impression that PB is valid one. Will check.

**Comment by wolfy-j** [comment]
https://developers.google.com/protocol-buffers/docs/reference/php-generated prefix PB is valid. Hm.

**Comment by wolfy-j** [comment]
Found it, they overwrite prefix :) 
https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/empty.proto

**Comment by wolfy-j** [comment]
Which is for Obecjtive C only... here we go again, divining into “that’s the only exception” scenario.

**Comment by zarianec** [comment]
Hmm, strange why this prefix is applied to php code if option is for Objective-C

**Comment by wolfy-j** [comment]
Per code 
https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/compiler/php/php_generator.cc#L237 there is absolutely no reference to GPB...

**Comment by wolfy-j** [comment]
LOL 
https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/compiler/php/php_generator.cc#L131

**Comment by wolfy-j** [comment]
Fixed in 1.0.3. Thank you!
